packages/@okta/vuepress-site/docs/guides/submit-app-prereq/main/index.md

```diff
@@ -31,7 +31,35 @@ Before using the [OIN Manager](https://oinmanager.okta.com)&nbsp;to submit your
+
+## OIN multi-tenancy
+
+Your app integration must support multi-tenancy to be listed in the public OIN catalog.
+
+What does this mean?
+
+Multi-tenancy in the OIN refers to the concept that as an ISV, you support several instances of your app, each with a unique credential system for your customers. An instance of an app that contains the infrastructure to support a group of users is considered a tenant. See [Tenants in Okta](/docs/guides/oin-sso-overview/#tenants-in-okta).
+
+You must provide a method for each of your customer tenants to uniquely connect to their Okta org. This allows your customers to find your app integration from the OIN catalog in their own Okta org. Then, they can instantiate the app integration with their unique tenant credentials, either with your support or on their own.
+
+#### SAML SSO multi-tenant example
+
+The following multi-tenant example demonstrates the scenario where your Okta app integration supports SAML SSO and you configure SSO for your customers:
+
+* Customer A and customer B have separate instances of your app. Each customer has their own set of users. Both customers use Okta as an IdP.
+* Customer A adds your integration to their Okta org and obtains the SAML metadata from the integration. They contact you to enable SSO for their users on your app and forward you the SAML metadata.
+* Similarly, customer B adds your integration to their Okta org and informs you that they want to enable SSO on your app. They provide you with their unique SAML metadata for the SSO configuration, which is obtained from their app integration in Okta.
+* You're responsible for configuring and enabling SSO for both customers A and B with the SAML 2.0 metadata that they've provided. Your app platform must allow for a separate credential system connection with Okta for each customer.
+
+#### OIDC SSO multi-tenant example
+
+The following multi-tenant example assumes that your Okta app integration supports OIDC SSO and that you offer a self-service portal for your customers:
+
+* Customer A and customer B have separate instances of your app. Both customers have their own set of users, and each uses Okta as an IdP.
+* Customer A instantiates an OIDC integration for your app in their Okta org. They obtain the Okta app integration client ID and secret. They then sign in to your app platform portal and set up SSO configuration with their client ID, client secret, and Okta domain. Customer A enables SSO to your app for their users, and doesn't require any external assistance.
+* Similarly, customer B instantiates your OIDC app integration in their Okta org. They obtain their unique client ID and secret from their Okta org. They then sign in to their account on your app platform. They use their client ID, client secret, and Okta domain to enable SSO without any assistance from you.
```

Comment:
This felt a little clunky but I presume it's an Acrolinx thing complaining about a long sentence.

Reply:
Yes, I was trying to please Acrolinx :-) 
I'll try to merge some of the sentences without annoying Acrolinx too much. lol